cfgov/v1/jinja2/v1/layouts/content-base.html

```diff
@@ -1,5 +1,9 @@
 {% extends 'v1/layouts/base.html' %}
 
+{% if page and not breadcrumb_items %}
+{% set breadcrumb_items = page.get_breadcrumbs(request) %}
+{% endif %}
+
```

Comment:
Could we move this chunk down below to above the `{% if breadcrumb_items ... %}` line? That would put it inside of a Jinja block, thus avoiding any potential confusion about when or if `{% set %}` statements are evaluated outside of blocks. It would also move it closer to where the breadcrumbs are actually used.

Reply:
https://github.com/cfpb/consumerfinance.gov/pull/7641/commits/8bba956fd05bbdfd916531ed85a6d3c2cbb6cd9f